eta/core/video.py

```diff
@@ -1229,20 +1246,28 @@ class FFmpegVideoSampler(FFmpeg):
     '''Class for sampling videos using ffmpeg.
 
     Example usage:
-        sampler = FFmpegVideoSampler(24)
-        sampler.run("/path/to/video.mp4", "/path/to/frames/%05d.png")
+        sampler = FFmpegVideoSampler(fps=10)
+        sampler.run("/path/to/video.mp4", "/path/to/sampled.mp4")
     '''
 
-    def __init__(self, fps=None, **kwargs):
+    DEFAULT_OUT_OPTS = [
+        "-c:v", "libx264", "-pix_fmt", "yuv420p",
+        "-preset", "medium", "-crf", "23", "-an",
```

Comment:
23 is a bit high, I think 20 or 21 will result in videos with less artifacts; if we need a default.